typings/custom-global/index.d.ts

```diff
@@ -1,5 +1,13 @@
 /// <reference types="webpack-env" />
 
+/**
+ * Generic typings for sass files.
+ */
+declare module '*.scss' {
```

Comment:
Downside of this is that it removes type-safety for styles that have already been built...

Reply:
I think we need a better way to get type safety than to rely on the build to produce the typings. Right now you get red squiggles without a build.